### PR TITLE
Add passwords recipe

### PR DESCRIPTION
**Pull request by cal-stytch**

### CODE REVIEW

File: components/Passwords/ResetPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import { registerWebAuthn, registerWebAuthnStart } from '../../lib/webAuthnUtils';
+import * as webauthnJson from '@github/webauthn-json';
```
Comment (chris-stytch) [inline components/Passwords/ResetPassword.tsx:3]: Some of these imports can be removed.

Comment (cal-stytch) [reply components/Passwords/ResetPassword.tsx:3]: Good eyes

File: pages/authenticate.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { useStytchUser, useStytch } from '@stytch/nextjs';
 
 const OAUTH_TOKEN = 'oauth';
 const MAGIC_LINKS_TOKEN = 'magic_links';
+const RESET_LOGIN = 'login';
```
Comment (chris-stytch) [inline pages/authenticate.tsx:7]: I think we're missing `reset_password` type.

Should be both `login` which equals the "EML escape hatch" in the UI and `reset_password` that punts you into the reset password flow.

**Review by chris-stytch — APPROVED** [review]

File: pages/authenticate.tsx (hunk 2 above)
Comment (cal-stytch) [reply pages/authenticate.tsx:7]: I have different redirect URLs, so resets just go to a completely different page. Reset logins go to the same page as a normal magic link/OAuth directs to

**Pull request merged**